src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: $trans-duration;
+
```

Comment:
if you use variable you should it more than one time if not dont use it

Reply:
fixed

Reply:
i used it in header and catalog blocks